gcc/config/riscv/riscv-ftypes.def

```diff
@@ -28,9 +28,21 @@ along with GCC; see the file COPYING3.  If not see
 
 DEF_RISCV_FTYPE (0, (USI))
 DEF_RISCV_FTYPE (1, (VOID, USI))
-DEF_RISCV_FTYPE (1, (USI, VOID_PTR)) //RISCV_USI_FTYPE_VOID_PTR
-DEF_RISCV_FTYPE (3, (SI, SI, SI, SI)) //RISCV_SI_FTYPE_SI_SI_SI
-DEF_RISCV_FTYPE (3, (USI, USI, USI, UQI)) //RISCV_USI_FTYPE_USI_USI_UQI
-DEF_RISCV_FTYPE (3, (SI, SI, SI, UQI)) //RISCV_SI_FTYPE_SI_SI_UQI
-DEF_RISCV_FTYPE (4, (USI, USI, USI, USI, UQI)) //RISCV_USI_FTYPE_USI_USI_USI_UQI
-DEF_RISCV_FTYPE (4, (SI, SI, SI, SI, UQI)) //RISCV_SI_FTYPE_SI_SI_SI_UQI
+DEF_RISCV_FTYPE (1, (USI, VOID_PTR))            //RISCV_USI_FTYPE_VOID_PTR
+DEF_RISCV_FTYPE (1, (USI, USI))                 //RISCV_USI_FTYPE_USI
+DEF_RISCV_FTYPE (2, (USI, USI, USI))            //RISCV_USI_FTYPE_USI_USI
+DEF_RISCV_FTYPE (2, (USI, USI, HI))             //RISCV_USI_FTYPE_USI_HI
+DEF_RISCV_FTYPE (2, (USI, USI, QI))             //RISCV_USI_FTYPE_USI_QI
+DEF_RISCV_FTYPE (2, (USI, USI, UHI))            //RISCV_USI_FTYPE_USI_UHI
+DEF_RISCV_FTYPE (2, (USI, USI, UQI))            //RISCV_USI_FTYPE_USI_UQI
+DEF_RISCV_FTYPE (3, (USI, USI, USI, USI))       //RISCV_USI_FTYPE_USI_USI_USI
+DEF_RISCV_FTYPE (3, (USI, USI, HI, USI))        //RISCV_USI_FTYPE_USI_HI_USI
+DEF_RISCV_FTYPE (3, (USI, USI, QI, USI))        //RISCV_USI_FTYPE_USI_QI_USI
+DEF_RISCV_FTYPE (3, (USI, USI, UHI, USI))       //RISCV_USI_FTYPE_USI_UHI_USI
+DEF_RISCV_FTYPE (3, (USI, USI, UQI, USI))       //RISCV_USI_FTYPE_USI_UQI_USI
+DEF_RISCV_FTYPE (3, (USI, USI, USI, QI))        //RISCV_USI_FTYPE_USI_USI_QI
+DEF_RISCV_FTYPE (3, (SI, SI, SI, SI))           //RISCV_SI_FTYPE_SI_SI_SI
+DEF_RISCV_FTYPE (3, (USI, USI, USI, UQI))       //RISCV_USI_FTYPE_USI_USI_UQI
+DEF_RISCV_FTYPE (3, (SI, SI, SI, UQI))          //RISCV_SI_FTYPE_SI_SI_UQI
+DEF_RISCV_FTYPE (4, (USI, USI, USI, USI, UQI))  //RISCV_USI_FTYPE_USI_USI_USI_UQI
+DEF_RISCV_FTYPE (4, (SI, SI, SI, SI, UQI))      //RISCV_SI_FTYPE_SI_SI_SI_UQI
```

Comment:
Note warning about missing newline at end of file! Check other files as well (looks to be all tests!).

Reply:
All fixed.